src/components/Card.component.jsx

```diff
@@ -0,0 +1,63 @@
+import styled from "styled-components";
+
+const Card = () => {
+  return (
+    <div>
+      <CardBox>
+        <Image src="/IMG_5696.JPG" alt="profile image"></Image>
+        <TextArea>
+          <NameArea>에코노베이션 26기 곽민준</NameArea>
+          <IntroduceArea>
+            안녕하세요! 리액트 스터디에 뒤늦게 참여하게 된 곽민준이라고 합니다.
+            리액트를 배워보기는 했지만 깊게 파고들어보지는 못했는데 좋은 기회
+            주셔서 감사합니다.
+            <br />
+            <br />
+            이번 스터디를 통해 많은 것을 얻어가고 추후에는 제가 누군가에게 도움
+            될 수 있도록 노력하겠습니다. 잘 부탁드립니다!
+          </IntroduceArea>
+        </TextArea>
+      </CardBox>
+    </div>
+  );
+};
+
+export default Card;
+
+const CardBox = styled.div`
+  width: 50rem;
+  height: 18rem;
+  box-shadow: 0 0.2rem 4rem rgba(0, 0, 0, 0.1);
+  border-radius: 0.5rem;
+  margin: 0 auto;
+  margin-top: 11rem;
```

Comment:
취향 차이일 수도 있겠지만 저는 아래와 같이 한 라인으로 정리하는 것을 선호합니다!
```suggestion
  margin: 11rem auto 0;
```

Reply:
아 이 부분은 한 라인으로 정리하는게 훨씬 깔끔해보입니다! 감사합니다